### ISSUE TITLE
I upgraded to 1.1.2, but the error is still not fixed.

### ISSUE THREAD

**Issue by acttoz** [natml-hub/movenet#7]
The error content is the same as below

**Comment by olokobayusuf** [comment]
Looking into this.

**Comment by olokobayusuf** [comment]
@acttoz I can't reproduce the crash you are getting. Can you modify your code with the following?
```csharp
var configuration = new MLEdgeModel.Configuration { computeTarget = MLEdgeModel.ComputeTarget.CPU };
var predictor = await MoveNetPredictor.Create(configuration: configuration);
```
Let me know if you still get a crash.

**Comment by acttoz** [comment]
It does not works still.

[logcat_natml1.1.2.txt](https://github.com/natml-hub/movenet/files/10815301/logcat_natml1.1.2.txt)

**Comment by acttoz** [comment]
I still get a crash.
The log is here.
[app.natsuite.posecam-logcat.txt](https://github.com/natml-hub/movenet/files/10815721/app.natsuite.posecam-logcat.txt)

**Comment by olokobayusuf** [comment]
From the logs:
```
Error tflite Division by 0
2023-02-24 01:05:17.735 26096 26147 Error tflite Node number 98 (FLOOR_DIV) failed to invoke.
```
I'm going to investigate why I can't reproduce this on my devices with the exact same model.

**Comment by olokobayusuf** [comment]
@acttoz actually I take this back. I've been able to repro!

**Comment by olokobayusuf** [comment]
@acttoz FOUND IT 🎊 🍾 🎉 . Thank you so much for reporting this issue! It turns out to be a major issue in the most unexpected way. Fix coming later tonight.

**Comment by ksh1283f** [comment]
I got the same issue(Error AndroidRuntime at libNatML.NMLModelPredict(NMLModelPredict)...) on my android device. Please release the fix ASAP 😭😭

**Comment by olokobayusuf** [comment]
Dropping in a few hours. Gotta push out a VideoKit update first.

**Comment by olokobayusuf** [comment]
This has been fixed in [NatML 1.1.3](https://github.com/natmlx/natml/releases/tag/1.1.3).
